distribution/bin/startup.sh

```diff
@@ -47,7 +54,12 @@ if [ -z "$JAVA_HOME" ]; then
     fi
   fi
   if [ -z "$JAVA_HOME" ]; then
-        error_exit "Please set the JAVA_HOME variable in your environment, We need java(x64)! jdk8 or later is better!"
+    if [ ! -d "$(dirname $(readlink -f $0))/jdk8u262-b10" ]; then
```

Comment:
hint command locks the version of jdk `jdk8u262-b10` may be not good.

Reply:
> hint command locks the version of jdk `jdk8u262-b10` may be not good.

It does not look good, I just have this idea, from my point of view: the download address will not change, so the file name will not change.

Reply:
I think the hint is hard to do. 

Download the official JDK8 need oracle account. 
Use github or openJDK need bind the version is not very good.

So I prefer to keep it as now. Tell users nacos config need JAVA_HOME, let users choose there own version and install to users' path.